### PR TITLE
Changed ping to netcat in Internet connection test

### PR DESCRIPTION
**Pull request by piosz**
Follow up #10285

cc @thockin @quinton-hoole

### CODE REVIEW

File: test/e2e/networking.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ var _ = Describe("Networking", func() {
 					{
 						Name:    contName,
 						Image:   "gcr.io/google_containers/busybox",
-						Command: []string{"ping", "-c", "3", "-w", "10", "google.com"},
+						Command: []string{"nc", "-w", "30", "google.com", "80"},
```
Comment (thockin) [inline test/e2e/networking.go]: I have had a bunch of problems with nc being flaky without a TTY.  Can you use curl or wget (I think busybox has wget but not curl)

Comment (quinton-hoole) [inline test/e2e/networking.go]: I think that if you use the -z flag, works without a TTY. But I don't feel strongly either way.  wget or curl are also fine.

  -z      Specifies that nc should just scan for listening daemons, without sending any data to them.  It is an error to use this option in
             conjunction with the -l option.

Comment (thockin) [inline test/e2e/networking.go]: Also lots of incompat across 'nc' implementations.

Comment (piosz) [inline test/e2e/networking.go]: There is no option -z in busybox version of netcat. I'll change to wget.

**Comment by piosz** [comment]
PTAL

**Comment by quinton-hoole** [comment]
LGTM.  Lets wait for the Jenkins GCE e2e to complete, and then I'll label ok-to-merge.

**Comment by quinton-hoole** [comment]
@k8s-bot Test this please.

**Comment by quinton-hoole** [comment]
Aargh! Jenkins is broken: jenkinsci/ghprb-plugin#117
Please merge anyway - this is low enough risk.
ok-to-merge

**Comment by mbforbes** [comment]
need one more... @davidopp ?

**Comment by mbforbes** [comment]
(Also shippable isn't happy...)

**Comment by davidopp** [comment]
LGTM

OK to merge once shippable succeeds.

**Pull request merged**